src/pages/settings/Payments/PaymentMethodList.js

```diff
@@ -155,23 +149,11 @@ function isPaymentMethodActive(actionPaymentMethodType, activePaymentMethodID, p
     return paymentMethod.accountType === actionPaymentMethodType && paymentMethod.methodID === activePaymentMethodID;
 }
 function PaymentMethodList(props) {
-    const {
-        actionPaymentMethodType,
-        activePaymentMethodID,
-        bankAccountList,
-        cardList,
-        fundList,
-        filterType,
-        network,
-        onPress,
-        payPalMeData,
-        shouldShowSelectedState,
-        selectedMethodID,
-        translate,
-    } = props;
+    const {actionPaymentMethodType, activePaymentMethodID, bankAccountList, fundList, filterType, network, onPress, payPalMeData, shouldShowSelectedState, selectedMethodID, translate} =
```

Comment:
non blocker: I can't remember if we formally added it, but we have a new style guide to de-structure props inside the function input.

```js
unction PaymentMethodList({actionPaymentMethodType,  activePaymentMethodID, ...})
```

Reply:
Good to know moving forward! Will merge this for now